app/src/main/java/org/onionshare/android/ui/MainViewModel.kt

```diff
@@ -66,13 +94,45 @@ class MainViewModel @Inject constructor(
             ).apply {
                 addFiles(filesReady.files)
             }
-            webserverManager.start(sendPage)
+            ensureActive()
```

Comment:
I guess this is one of the places where we'll need to think about cleaning up zip files.

Reply:
Should we do a clean up zip files ticket where we go through the code and handle this, or is the TODO in the code enough for now?

Reply:
Doing both sounds good to me, but I'm more prone to forgetting things than you. :-)

Reply:
I created #10 for this.